src/Services/ConsultationService.php

```diff
@@ -248,22 +223,32 @@ public function forCurrentUserResponse(ListConsultationsRequest $listConsultatio
         $consultations
             ->select(
                 'consultations.*',
-                'consultation_terms.order_item_id',
-                'consultation_terms.executed_status',
-                'consultation_terms.executed_at',
+                'consultation_user.id as cuid',
+                'consultation_user.executed_status',
+                'consultation_user.executed_at',
             )
-            ->leftJoin('consultation_terms', 'consultation_terms.consultation_id', '=', 'consultations.id');
+            ->leftJoin('consultation_user', 'consultation_user.consultation_id', '=', 'consultations.id');
         $consultationsCollection = ConsultationSimpleResource::collection($consultations->paginate(
             $listConsultationsRequest->get('per_page') ??
             config('escolalms_consultations.perPage', ConstantEnum::PER_PAGE)
         ));
         ConsultationSimpleResource::extend(function (ConsultationSimpleResource $consultation) {
             return [
-                'order_item_id' => $consultation->order_item_id,
+                'consultation_user_id' => $consultation->cuid,
                 'executed_status' => $consultation->executed_status,
                 'executed_at' => $consultation->executed_at,
             ];
         });
         return $consultationsCollection;
     }
+
+    public function attachToUser(Consultation $consultation, User $user): void
```

Comment:
Look at how attaching Products and Productables is currently handled in Cart package
You need to think how to handle `quantity`, as Cart package expects increasing (and maybe even decreasing) of quantity of owned Products and Productables to be possible; unless you want to call this in `for($i=0; $i<$quantity; $i++)` loop xd

https://github.com/EscolaLMS/Cart/blob/818912072824e82079de24ad08298ef0dddc05b8/src/Services/ProductService.php#L313